src/main/java/by/itechart/api/controller/UserControllerInfo.java

```diff
@@ -3,16 +3,33 @@
 import by.itechart.api.dto.CreateUserDTO;
 import by.itechart.api.dto.UpdateUserDTO;
 import by.itechart.api.dto.UserDTO;
+import by.itechart.api.util.annotation.ApiPageable;
 import io.swagger.annotations.*;
 import io.swagger.v3.oas.annotations.Operation;
 import io.swagger.v3.oas.annotations.security.SecurityRequirement;
 import org.springframework.data.domain.Pageable;
 import org.springframework.http.ResponseEntity;
 import org.springframework.security.core.Authentication;
+import springfox.documentation.annotations.ApiIgnore;
 
 import java.util.List;
 
 public interface UserControllerInfo {
+
+    @Operation(security = @SecurityRequirement(name = "basicAuth"))
+    @ApiOperation(value = "Get all active users", notes = "This method will return all active users", authorizations = {
+            @Authorization(value = "basic", scopes = {@AuthorizationScope(scope = "getUsers", description = "getAllUsers")})
+    })
```

Comment:
Why are you duplicating this info about auth method?

Reply:
In `@Operation` annotation you mentioned that this method requires basic authentication, and then in `@ApiOperation` you also mentioned `@Authorization(value="basic")`

Reply:
Okay, now I got it, thanks! :)